### PR TITLE
Added a developer

### PR DESCRIPTION
**Pull request by samyakbvs**

### CODE REVIEW

**Review by 0x48piraj — COMMENTED** [review]
`</section>` is properly not indented.

**Comment by gabru-md** [comment]
@samyakbvs Can you provide us with a link to preview your changes?

**Comment by samyakbvs** [comment]
https://samyakbvs.github.io/jainbvs-fossasia.org/team 
@gabru-md

**Comment by gabru-md** [comment]
All the images have a definite spacing/margin between the image and names. But here it seems to be different. Please have a look at it. :)

spacing

**Review by gabru-md — CHANGES_REQUESTED** [review]
added a comment. please see and try to resolve

**Comment by samyakbvs** [comment]
how should i do it?

**Comment by samyakbvs** [comment]
Done @gabru-md

**Review by gabru-md — COMMENTED** [review]
Great!

**Review by gabru-md — APPROVED** [review]
Approved

**Comment by samyakbvs** [comment]
Is it merge ready?

**Comment by gabru-md** [comment]
@harshit98 merge ?

**Review by ParthS007 — APPROVED** [review]

**Comment by samyakbvs** [comment]
@harshit98 Please see

**Review by 0x48piraj — APPROVED** [review]
LGTM!

File: team.html [hunk 1]
```diff
@@ -1451,603 +1467,577 @@ <h1>Enablement Team</h1>
...
-						<ul class="contact-methods">
-							<li><i class="icon pe-7s-mail"></i><span><a id="mailAdd" href="mailto:fossasia@googlegroups.com" target="_self" style="color:#777777">fossasia@googlegroups.com</a></span></li>
-							<li><i class="icon pe-7s-phone"></i><span>+65 83435672</span></li>
-							<li><i class="icon pe-7s-map-marker"></i><span>Ms. DANG Hong Phuc</span></li>
-							<li><span>FOSSASIA</span></li>
-							<li><span>6 Eu Tong Sen Street, #5-07 The Central, Singapore 059817</span></li>
-						</ul>
-					</div>
-				</div>
-
```
Comment (abishekvashok) [inline team.html]: Please keep the current intendation as it is. Was'nt it right?

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by samyakbvs** [comment]
@abishekvashok that code isn't working in my link so I edited the code

**Comment by samyakbvs** [comment]
And now the section is fine

**Review by abishekvashok — APPROVED** [review]
ack https://github.com/fossasia/fossasia.org/pull/226/commits/4c54f17af41f3d29c9b9a9fb2081037a82160783

**Comment by samyakbvs** [comment]
What does ack stand for?

**Comment by samyakbvs** [comment]
What does ack stand for?

**Comment by harshit98** [comment]
@samyakbvs 
- Always provide a demo link, so that it is easy for mentors to review the pull request fast. (Your demo link: https://samyakbvs.github.io/jainbvs-fossasia.org/team)
- A short abstract describing the fix.
- Keep the current indentation. Don't change it.

**Comment by samyakbvs** [comment]
@harshit98 the previous code wasn't working so I had to change it thats why the indentation is different

**Comment by samyakbvs** [comment]
And also I did provide the preview link if you scroll upwards you will find it

**Comment by 0x48piraj** [comment]
As I said you at some other issue, Always provide the changes you made :+1:

**Comment by harshit98** [comment]
Okay. @samyakbvs try to keep the old indented code. We have to maintain code quality as well.

**Comment by samyakbvs** [comment]
now is it fine @harshit98

**Comment by harshit98** [comment]
Make a new PR @samyakbvs if you find any difficulties in indenting the code. I'll prefer the old indented code.

**Comment by harshit98** [comment]
Ok. PR looks good to me!

**Review by harshit98 — APPROVED** [review]

**Review by isuruAb — APPROVED** [review]
👍

**Comment by samyakbvs** [comment]
Somebody pls merge this?

**Review by codedsun — APPROVED** [review]

**Review by the-codingninja — APPROVED** [review]

**Comment by samyakbvs** [comment]
pls merge guys

**Comment by samyakbvs** [comment]
@mariobehling

**Pull request merged**